packages/account/src/Sections/Security/LoginHistory/__test__/login-history.spec.tsx

```diff
@@ -0,0 +1,170 @@
+import React from 'react';
+import { screen, render, waitFor } from '@testing-library/react';
+import { WS, isMobile } from '@deriv/shared';
+import { StoreProvider, mockStore } from '@deriv/stores';
+import LoginHistory from '../login-history';
+import { getLoginHistoryFormattedData } from '../../../../../../utils/src/getLoginHistoryFormattedData';
```

Comment:
Didn't `@deriv/api` as path work here?

Reply:
you mean `import { getLoginHistoryFormattedData } from '@deriv/api';`?
getLoginHistoryFormattedData is from utils.